src/ui/public/notify/banners/global_banner_list.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiPanel,
+} from '@elastic/eui';
+
+/**
+ * GlobalBannerList is a list of "banners". A banner something that is displayed at the top of Kibana that may or may not disappear.
+ *
+ * Whether or not a banner can be closed is completely up to the author of the banner. Some banners make sense to be static, such as
+ * banners meant to indicate the sensitivity (e.g., classification) of the information being represented.
+ *
+ * Banners are currently expected to be <EuiCallout /> instances, but that is not required.
+ *
+ * @param {Array} banners The array of banners represented by objects in the form of { id, component }.
+ */
+export class GlobalBannerList extends Component {
+
+  static propTypes = {
+    banners: PropTypes.array,
+    subscribe: PropTypes.func,
+  };
+
+  static defaultProps = {
+    banners: [],
+  };
+
+  constructor(props) {
+    super(props);
+
+    if (this.props.subscribe) {
+      this.props.subscribe(() => this.forceUpdate());
```

Comment:
Now that I read this, `subscribe` does read strangely but I'll be darned if I can think of anything better.